tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1672,6 +1672,25 @@ public function getJsonDecodedResponseBodyContent(ResponseInterface $response =
 		return null;
 	}
 
+	/**
+	 * @Then the ocs JSON data of the response should match
```

Comment:
@PrajwolAmatya @SagarGi  watch out for each other's same-step implementation https://github.com/owncloud/ocis/pull/5651
https://github.com/owncloud/ocis/blob/8e7fcf862b789ee0bc8102dc0fba188782ff40f0/tests/acceptance/features/bootstrap/FeatureContext.php#L1695-L1713